app/routes/api/jokes/$id/rate.ts

```diff
@@ -44,5 +48,20 @@ export const action: ActionFunction = async ({ request, params }) => {
 
   await ratingService.upsertRating(rating);
 
+  const ratings = await db.rating.groupBy({
```

Comment:
I figured we could send the updated ratings for the given joke over pusher so that the UI could just update it in place without another network call.